crates/glue/tests/test_glue_cli.rs

```diff
@@ -198,83 +197,76 @@ mod glue_cli_run {
         assert_eq!(all_fixtures, &mut VecSet::default());
     }
 
-    fn generate_glue_for<'a, I: IntoIterator<Item = &'a str>>(
-        platform_dir: &'a Path,
-        args: I,
-    ) -> Out {
-        let platform_module_path = platform_dir.join("platform.roc");
-        let glue_dir = platform_dir.join("test_glue");
-        let fixture_templates_dir = platform_dir
-            .parent()
-            .unwrap()
-            .parent()
-            .unwrap()
-            .join("fixture-templates");
-
-        // Copy the rust template from the templates directory into the fixture dir.
-        dircpy::CopyBuilder::new(fixture_templates_dir.join("rust"), platform_dir)
-            .overwrite(true) // overwrite any files that were already present
-            .run()
-            .unwrap();
-
-        // Delete the glue file to make sure we're actually regenerating it!
-        if glue_dir.exists() {
-            fs::remove_dir_all(&glue_dir)
-                .expect("Unable to remove test_glue dir in order to regenerate it in the test");
-        }
-
-        let rust_glue_spec = fixture_templates_dir
-            .parent()
-            .unwrap()
-            .parent()
-            .unwrap()
-            .join("src")
-            .join("RustGlue.roc");
-
-        // Generate a fresh test_glue for this platform
-        let parts : Vec<_> =
-            // converting these all to String avoids lifetime issues
-            std::iter::once("glue".to_string()).chain(
-                args.into_iter().map(|arg| arg.to_string()).chain([
-                    rust_glue_spec.to_str().unwrap().to_string(),
-                    glue_dir.to_str().unwrap().to_string(),
-                    platform_module_path.to_str().unwrap().to_string(),
-                ]),
-            ).collect();
-        let glue_out = run_glue(parts.iter());
-
-        if has_error(&glue_out.stderr) {
-            panic!(
-                "`roc {}` command had unexpected stderr: {}",
-                parts.join(" "),
-                glue_out.stderr
-            );
-        }
-
-        assert!(glue_out.status.success(), "bad status {glue_out:?}");
-
-        glue_out
-    }
+    // fn generate_glue_for<'a, I: IntoIterator<Item = &'a str>>(
+    //     platform_dir: &'a Path,
```

Comment:
TODO -- check if we can probably remove all of this, there shouldn't be any test platforms remaining that require glue gen.